src/renderer/register/components/RegisterForm/index.js

```diff
@@ -1,17 +1,32 @@
 import { compose, withState } from 'recompose';
-import { progressValues } from 'spunky';
+import { withActions, progressValues } from 'spunky';
 
-import withAlert from 'shared/hocs/withAlert';
+import withImmediateReset from 'shared/hocs/withImmediateReset';
+import withLoadingProp from 'shared/hocs/withLoadingProp';
+import withErrorToast from 'shared/hocs/withErrorToast';
+import pureStrategy from 'shared/hocs/strategies/pureStrategy';
 import withProgressChange from 'shared/hocs/withProgressChange';
 
 import RegisterForm from './RegisterForm';
 import createAccountActions from '../../actions/createAccountActions';
 
 const { FAILED } = progressValues;
 
+const mapAccountActionsToProps = (actions) => ({
+  onRegister: ({ passphrase, passphraseConfirmation }) => {
+    return actions.call({ passphrase, passphraseConfirmation });
+  }
+});
+
 export default compose(
+  withImmediateReset(createAccountActions),
```

Comment:
Is this a new one? Didn't see it passing by :P

Reply:
No, I just moved it from `Register/index.js` here because it made more sense & helped with getting the container test to work as expected.